pom.xml

```diff
@@ -4,7 +4,7 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>groupId</groupId>
+    <groupId>org.eclipse.iofog</groupId>
     <artifactId>ioElementCoreNetworking</artifactId>
     <version>1.0-SNAPSHOT</version>
```

Comment:
- i thinks snapshot can be removed, and 
- probably version 0.1 will be enough for now
- actifactId can be more java style like : core-networking

Reply:
did you build jar file to check that build passes ?